presentation/common/src/test/LocalizationHelper.test.ts

```diff
@@ -0,0 +1,235 @@
+      const testCategory = createTestCategoryDescription({ description: "@namespace:LocalizedDescription@" });
+      const field = createTestSimpleContentField({ category: testCategory });
+      const content = new Content(createTestContentDescriptor({ fields: [field], categories: [testCategory] }), [contentItem]);
+      localizationHelper.getLocalizedContent(content);
+      expect(content.descriptor.categories[0].description).to.be.eq("LocalizedDescription");
+    });
+
+    it("translates element properties label", () => {
+      const elementProperties = localizationHelper.getLocalizedElementProperties({ class: "class", label: "@namespace:LocalizedLabel@", id: "id", items: {} });
+      expect(elementProperties.label).to.be.eq("LocalizedLabel");
+    });
+
+  });
+
+  describe("getLocalizedLabelDefinition", () => {
+
+    it("translates labelDefinition", () => {
+      const labelDefinition = createRandomLabelDefinition();
+      labelDefinition.rawValue = "@namespace:LocalizedValue@";
+      localizationHelper.getLocalizedLabelDefinition(labelDefinition);
+      expect(labelDefinition.rawValue).to.be.eq("LocalizedValue");
+    });
+
+    it("translates labelDefinition with composite value", () => {
+      const compositeValue = createRandomLabelCompositeValue();
+      compositeValue.values.forEach((value) => {
+        value.rawValue = "@namespace:LocalizedValue@";
+      });
+      const labelDefinition: LabelDefinition = {
+        displayValue: "Display",
+        rawValue: compositeValue,
+        typeName: LabelDefinition.COMPOSITE_DEFINITION_TYPENAME,
+      };
+      localizationHelper.getLocalizedLabelDefinition(labelDefinition);
+      compositeValue.values.forEach((value) => {
+        expect(value.rawValue).to.be.eq("LocalizedValue");
+      });
+
+    });
+
+    it("does not translate non string value", () => {
+      const labelDefinition: LabelDefinition = {
+        displayValue: "10",
+        rawValue: 10,
+        typeName: "int",
+      };
+      localizationHelper.getLocalizedLabelDefinition(labelDefinition);
+      expect(labelDefinition.rawValue).to.be.eq(10);
+    });
+
+  });
+
+  describe("getLocalizedLabelDefinitions", () => {
+
+    it("translates labelDefinitions", () => {
+      const labelDefinitions = [createRandomLabelDefinition(), createRandomLabelDefinition()];
+      labelDefinitions.forEach((labelDefinition) => labelDefinition.rawValue = "@namespace:LocalizedValue@");
+      localizationHelper.getLocalizedLabelDefinitions(labelDefinitions);
+      labelDefinitions.forEach((labelDefinition) => {
+        expect(labelDefinition.rawValue).to.be.eq("LocalizedValue");
+      });
+    });
+
+  });
+
+});
+
+describe("getLocalizedStringEN", () => {
+  let localizationHelper: LocalizationHelper;
+
+  beforeEach(() => {
+    localizationHelper = new LocalizationHelper({ getLocalizedString: getLocalizedStringEN });
+  });
+
+  it("translates from RulesEngine", () => {
```

Comment:
Maybe add all existing key in the files to the test? I think there are not that many localizable string in these files.